docs/shared/reference/releases.md

```diff
@@ -0,0 +1,39 @@
+# Releases
+
+Major Nx versions are released as the _latest_ every six months, typically around April and October.
+After each major version release, the _previous_ major version moves to long-term support (LTS) for 12 months, after
+which it becomes unsupported.
+Each major version has 18 total months of support from when it is first released to when it falls out of LTS.
+Workspaces should only use major versions that are under LTS.
+
+## Supported Versions
+
+The following are the currently supported versions of Nx.
+
+| Version | Release Date |
+| ------- | ------------ |
+| v19     | 2024-05-06   |
+| v18\*   | 2024-02-03   |
+| v17     | 2023-10-19   |
+| v16     | 2023-04-27   |
+
+**\*Note:** v18 is a special release and does not fit into the normal release cycle. Thus, v16 continues to be supported
+according to schedule.
+
+### LTS Fixes
+
+LTS fixes are generally considered if they address a security vulnerability or a critical bug that impedes the use of
+that version of Nx.
+
+## Deprecation Policy
+
+When the Nx team intends to remove an API or feature, it will be marked as _deprecated_. The deprecated API will remain
+functional for two major versions (or one year), after which they will be removed.
+
```

Comment:
I think an explicit example may help here. Something along the lines of... If something is deprecated in `16.2.0`, it will still work as expected for the entirety of Nx 17. When Nx 18 is released, the feature would be removed.

Also, add more description stating that... these deprecations may show up in the documentation, as warnings in console output, and/or TS Doc.